lib/ukrand/include/sys/random.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef _SYS_RANDOM_H
```

Comment:
Double underscores for guards: `__SYS_RANDOM_H__`... oh but this is from the old library... how did Github UI mess this up showing it as added lines :thinking:. Oh, it's because you changed the license.